### PR TITLE
US123175: [Engagement] Card Layout Updates

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US123175](https://rally1.rallydev.com/#/23525809118d/iterationstatus?detail=%2Fuserstory%2F462659131420&sharedViewId=328666453148&view=dc9e7043-1cf6-486d-a4aa-3a212976e789&fdp=true?fdp=true): [Engagement] Card Layout Updates

5-8 cards layout is outside the scope of this PR.

* Functional Testing 
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility - no changes were made that could affect accessibility
  * [x] Responsive
  * [x] Unit tests
  * [x] Test cases
    * Changes does not break Settings page
    * Cards correspond design for
       * 4 visible cards
       * 3 visible cards
       * 2 visible cards
       * 1 visible cards
* Security, devops, perf: N/A
* UX and docs
  * [x] Demo to Kevin

Pic 1

image

Pic 2

image

Pic 2-responsive

image

Pic 3

image

Pic 3-responsive 

image

Pic 4

image

### CODE REVIEW

**Comment by Vitalii-Misechko** [comment]
FYI @anhill-D2L @MykolaGalian @rohitvinnakota @hyehayes @bemailloux

File: components/summary-cards-container.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+		return html`<d2l-insights-discussion-activity-card .data="${this.data}" ?wide="${wide}" ?tall="${tall}" ?skeleton="${this.skeleton}"></d2l-insights-discussion-activity-card>`;
+	}
+
+	_lastAccessCard({ wide, tall }) {
+		return html`<d2l-insights-last-access-card .data="${this.data}" ?wide="${wide}" ?tall="${tall}" ?skeleton="${this.skeleton}"></d2l-insights-last-access-card>`;
+	}
+
+	_handleResize() {
+		// causes rendering only if the media-query breakpoint in _isSmallScreen is triggered
+		this._screenSize = this._getScreenSize();
```
Comment (johngwilkinson) [inline components/summary-cards-container.js:178]: Nice!

**Review by johngwilkinson — APPROVED** [review]

File: engagement-dashboard.js [hunk 2]
```diff
@@ -106,6 +104,13 @@ class EngagementDashboard extends Localizer(MobxLitElement) {
 			heading3Styles,
 			css`
 				:host {
+					--d2l-insights-engagement-small-card-height: 154px;
+					--d2l-insights-engagement-small-card-width: 291px;
+					--d2l-insights-engagement-big-card-width: calc(var(--d2l-insights-engagement-small-card-width) * 2 + var(--d2l-insights-engagement-card-margin-right)); /* 594px; */
+					--d2l-insights-engagement-big-card-height: calc(var(--d2l-insights-engagement-small-card-height) * 2 + var(--d2l-insights-engagement-card-margin-top));	/* 318px; */
+					--d2l-insights-engagement-card-margin-top: 10px;
+					--d2l-insights-engagement-card-margin-right: 12px;
```
Comment (NicholasHallman) [inline engagement-dashboard.js:112]: Nice!

**Review by NicholasHallman — APPROVED** [review]

**Comment by Vitalii-Misechko** [comment]
Requested re-approval due to not reviewed unit test.

**Review by johngwilkinson — APPROVED** [review]

**Review by NicholasHallman — APPROVED** [review]

**Pull request merged**